src/firebaseRef.js

```diff
@@ -0,0 +1,49 @@
+(function() {
+  "use strict";
+
+  function FirebaseRef() {
+    this.urls = null;
+    this.registerUrl = function registerUrl(urlOrConfig) {
+
+      if (typeof urlOrConfig === 'string') {
+        this.urls = {};
+        this.urls.default = urlOrConfig;
+      }
+
+      if (angular.isObject(urlOrConfig)) {
+        this.urls = urlOrConfig;
+      }
+
+    };
+
+    this.$$checkUrls = function $$checkUrls(urlConfig) {
+      if (!urlConfig) {
+        return new Error('No Firebase URL registered. Use firebaseRefProvider.registerUrl() in the config phase. This is required if you are using $firebaseAuthService.');
+      }
+    };
+
+    this.$$createRefsFromUrlConfig = function $$createMultipleRefs(urlConfig) {
+      var error = this.$$checkUrls(urlConfig);
+      if (error) { throw error; }
+      var defaultUrl = urlConfig.default;
+      var defaultRef = new Firebase(defaultUrl);
+      delete urlConfig.default;
+      angular.forEach(urlConfig, function(value, key) {
+        if (!defaultRef.hasOwnProperty(key)) {
+          defaultRef[key] = new Firebase(value);
```

Comment:
After reviewing it, I agree.

I think modifying the Firebase ref can open up a can of worms.